sensor/DNSCryptPlugin.js

```diff
@@ -166,7 +169,7 @@ class DNSCryptPlugin extends Sensor {
   // global on/off
   async globalOn() {
     this.adminSystemSwitch = true;
-    await dc.restart();
+    //await dc.restart();
     await this.applyAll();
```

Comment:
I think it looks okay to check the local file every time, even including globalOn